pkgs/by-name/sp/sparkle/package.nix

```diff
@@ -0,0 +1,46 @@
+{
+  stdenvNoCC,
+  fetchurl,
+  lib,
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "sparkle";
+  version = "2.6.4";
+
+  src = fetchurl {
+    url = "https://github.com/sparkle-project/Sparkle/releases/download/${finalAttrs.version}/Sparkle-${finalAttrs.version}.tar.xz";
+    hash = "sha256-UGEqBgOKvJMfFgEdeQO4Mmo2LBB02rzLcYQEzo5YXws=";
+  };
```

Comment:
Why not fetchFromGitHub?

Reply:
Because I am fetching the prebuilt binary from GH releases instead of building from source. I unfortunately couldn't get `xcodebuild` and `xcbuildHook` to work (`xcbuild` just segfaults, and I don't know a way to get normal `xcodebuild` in the sandbox 😕).